Source/BroncoDrome/StageActors/AIActor.h

```diff
@@ -34,9 +34,10 @@ struct FDifficultyParameters {
   // constructor
   FDifficultyParameters() {
     difficulty = FName(TEXT("Medium"));
-    
   }
-
+  void decrementDifficulty() {
+    damageMod = healthMod = fireRateMod -= 0.1;

```

Comment:
Are you trying to decrement the difficulty for the player or the AI? From what I can tell, this mod would be applied to the player runner. I don't know what the damage/health/firerate mods do, but if they are at 0.9, wouldn't that mean that they are less effective and therefore makes it harder for the player? Or am I misunderstanding?